ui/scss/component/_videojs.scss

```diff
@@ -36,28 +36,45 @@ $control-bar-icon-size: 0.8rem;
   // Progress Bar
   .vjs-progress-control.vjs-control {
     position: absolute;
-    display: flex;
+    display: block;
     align-items: center;
     width: 100%;
     height: 1rem;
     bottom: 90%;
+    max-width: 96.5%;
+    margin: 1px 23px !important;
```

Comment:
When docked to the side (i.e. small layout), the progress bar is no longer symmetrical

<img width="329" alt="image" src="https://user-images.githubusercontent.com/64950861/169065595-2624be0a-53f5-4faf-8fe5-ca9980a3b36f.png">